src/Pact/Types/Term.hs

```diff
@@ -253,6 +254,26 @@ instance Semigroup Gas where
 instance Monoid Gas where
   mempty = 0
 
+newtype MilliGas
```

Comment:
A sentence or two about why `DecimalRaw Int64` wouldn't work (where `Gas n ~ Decimal 0 n` and `MilliGas n ~ Decimal 3 n`) and thus why two separate `Gas` and `MilliGas` are needed would be very useful for a future maintainer, I think.

Reply:
Do you mean `DecimalRaw` from `Data.Decimal`? Because we don't get to have fixed precision with it.

I can add a comment why though, sure.